src/docs/sdk/overview.mdx

```diff
@@ -229,6 +230,24 @@ The following additional headers are permitted as per CORS policy:
 - `content-encoding`
 - `transfer-encoding`
 
+### User Agent
+
+All SDKs are expected to report their name and version via the `user-agent` header. 
+The following format should be used (unless required or recommended differently by the platform, 
+e.g. for browser SDKs):
```

Comment:
I assume this is true?

```suggestion
e.g. for browser SDKs, where the user agent header must be set by the browser itself):
```

Reply:
confirmed by @Lms24. I believe this is the relevant `Transport` impl which doesn't set a `user-agent`:

https://github.com/getsentry/sentry-javascript/blob/ee4091e8850dac6f6cd8970e814f2c7f74d1a74f/packages/types/src/transport.ts#L4

Reply:
correct! Also, [this code](https://github.com/getsentry/sentry-javascript/blob/ee4091e8850dac6f6cd8970e814f2c7f74d1a74f/packages/browser/src/transports/fetch.ts/#L24-L42) shows that we only add headers additionally to the request that are specified by users in `Sentry.init({transportOptions: headers: {...}})`. So if users want to add headers to the request we'll do that but we don't add any by default.

Reply:
@Lms24 what are we doing for server side JS?

Reply:
@cleptric Good point, we [don't set](https://github.com/getsentry/sentry-javascript/blob/ee4091e8850dac6f6cd8970e814f2c7f74d1a74f/packages/node/src/transports/http.ts/#L127-L138) a user-agent either.